suites/message/message_application.go

```diff
@@ -116,5 +120,11 @@ func TestMessageApplicationEdgecases(t *testing.T, factory state.Factories) {
 			}, chain.Nonce(1), chain.Value(big_spec.Zero())),
 			types.MessageReceipt{ExitCode: exitcode.ErrIllegalArgument, ReturnValue: drivers.EmptyReturnValue, GasUsed: abi_spec.NewTokenAmount(1_000_000)},
 		)
+
+		// assert that the entire gas limit is deducted
+		gasReward := big.NewInt(td.MessageProducer.DefaultGasLimit())
+		gasReward = gasReward.Mul(gasReward, td.MessageProducer.DefaultGasPrice().Int)
+		senderBalance.Sub(senderBalance.Int, gasReward)
+		td.AssertBalance(sender, senderBalance)
```

Comment:
I left a comment about this here: https://github.com/filecoin-project/chain-validation/pull/104#discussion_r393815651. I would be in favor of closing this in favor of #104 since this appears to be duplicated from there.

Reply:
Yup, will do.